src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborConformanceLevel.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace System.Security.Cryptography.Encoding.Tests.Cbor
+{
+    internal enum CborConformanceLevel
+    {
+        Lax = 0,
+        Strict = 1,
+        Rfc7049Canonical = 2,
+        Ctap2Canonical = 3,
+    }
+
+    internal static class CborConformanceLevelHelpers
+    {
+        public static void Validate(CborConformanceLevel conformanceLevel)
+        {
+            if (!Enum.IsDefined(typeof(CborConformanceLevel), conformanceLevel))
```

Comment:
Since you're in the same assembly there's not a "this can give surprising results when new values get added" reason against Enum.IsDefined; but there's still a "it loads a lot of reflection data, so it's more expensive than it looks" reason against it.  The recommended approach is `if (conformanceLevel < CborConformanceLevel.Lax || conformanceLevel > CborConformanceLevel.Ctap2Canonical)`

```suggestion
            if (conformanceLevel < CborConformanceLevel.Lax ||
                conformanceLevel > CborConformanceLevel.Ctap2Canonical)
```